src/_guides/language/sound-problems.md

```diff
@@ -63,6 +63,12 @@ see [Runtime errors](#common-errors-and-warnings).
 error • The <member> '...' isn't defined for the class '...' • undefined_<member>
 ```
 
+These errors can appear under the following conditions:
+
+- A variable is statically known to be some supertype but the code assumes a subtype.
```

Comment:
but -> , but

(This was there before, but the punctuational correctness of the next item you added made me notice that this one was punctuated incorrectly. :)